### PR TITLE
refactor(RBAC): RHICOMPL-3690 configure rbac without url splitting

### PR DESCRIPTION
**Pull request by RoamingNoMaD**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: config/initializers/0_clowder-config.rb [hunk 1]
```diff
@@ -75,7 +82,10 @@ def build_endpoint_url(endpoint, config)
       remediation_updates: config.kafka_topics&.dig('platform.remediation-updates.compliance', 'name'),
       notifications: config.kafka_topics&.dig('platform.notifications.ingress', 'name')
     },
-    rbac_url: rbac_url,
+    rbac: {
+      host: "#{rbac_hostname}:#{rbac_port}",
+      scheme: "#{rbac_scheme}"
```
Comment (skateman) [inline config/initializers/0_clowder-config.rb:87]:
```suggestion
      scheme: rbac_scheme
```

File: config/initializers/0_clowder-config.rb [hunk 2]
```diff
@@ -25,7 +25,14 @@ def build_endpoint_url(endpoint, config)
 
   # RBAC
   rbac_config = config.dependency_endpoints.dig('rbac', 'service')
-  rbac_url = build_endpoint_url(rbac_config, config)
+  rbac_hostname = rbac_config.hostname
```
Comment (skateman) [inline config/initializers/0_clowder-config.rb:28]: Let's name it `rbac_host` for consistency. And maybe move the port into this variable as well. See below the reasoning.

**Review by skateman — APPROVED** [review]
The Seal of Approval

**Review by romanblanco — CHANGES_REQUESTED** [review]
@RoamingNoMaD `development.yml` and `production.yml` also need an update

**Comment by skateman** [comment]
> @RoamingNoMaD `development.yml` and `production.yml` also need an update

Not really, those use clowder and these values aren't even consumed at all. Eventually in the next ticket we will drop the test.yml as well, at least the clowder-related values by clowderizing the test environment. So no further changes are needed for now.

**Review by romanblanco — APPROVED** [review]

**Pull request merged**